src/core/qgis.h

```diff
@@ -5613,6 +5613,27 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( DocumentationBrowser )
 
+    /**
+     * Action to be performed by the mouse handles
+     *
+     * \since QGIS 3.42
+     */
+    enum class MouseHandlesAction : int
```

Comment:
You'll need to add the python compatibility code to this (ie SIP_MONKEYPATCH_SCOPEENUM_UNNEST), to maintain existing API.

See eg https://github.com/qgis/QGIS/blob/c61ed9864740be7aa92bb3e8bd39ed80a8e9b9c6/src/core/qgis.h#L508

Reply:
Is it really necessary? QgsGraphicsViewMouseHandles is not part of the PyQGIS API

Reply:
You're right :+1: